### ISSUE TITLE
Mulitsite compatibility 

### ISSUE THREAD

**Issue by arsors** [KaufmannDigital/KaufmannDigital.CookieConsent#1]
If you have created several pages with their own domain in the Neos backend, the plugin determines the slug of the stored node and places it behind the current domain.
Example: _a.com/privacy-policy_ and _b.com/privacy-policy_

The plugin ignores the fact that there is no privacy policy page at b.com. But if you create a site at b.com that has the same slug as a.com it works fine.

However, I think it would make more sense if you could specify a separate node for each deposited domain; or a global node can be deposited (if no node was specified). And while you're at it, a multi-language function wouldn't be wrong either.

I could imagine the following for implementation:
```
KaufmannDigital:
  CookieConsent:
    policyPageNode: 'c7a91aa8-fbac-4c24-8326-102eb7307180' #Global Identifier
    globalRootName: 'gobal-rootnodename' #Global Rootnode name, to get the right domain even on other domain which have no given privacy policy node
    sites: #If there are multi sites
      rootnodename: #Give the specific rootnode name
        policyPageNode: 'Identifier-8326-102eb7307180' #Specific Identifier for de and en
          language: ['de','en']
        policyPageNode: 'Identifier-8273-102eb7307180' #Specific Identifier for fr
          language: ['fr']
      otherrootnodename: #Give the specific rootnode name
        policyPageNode: [...]
```
That's just my first idea.

If nothing is specified under _sites:_ this root node should use the global privacy policy. If nothing is defined in _language:_ (but content dimension has been created for languages) the default language should be used as fallback.

**Comment by Nguonchhay** [comment]
Hi @taeq ,

I have one merge request to fix your above problem (https://github.com/KaufmannDigital/KaufmannDigital.CookieConsent/pull/4). If you have time, please help to test that merge request. If it works, please give a plus to notify the owner.

Thanks.

**Comment by Nikdro** [comment]
As @Nguonchhay said, this is solved by #4.

Thanks for the idea @taeq and the PR @Nguonchhay.
